idea/idea-fir/src/org/jetbrains/kotlin/idea/intentions/fir/AddAccessorsIntention.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions.fir
+
+import com.intellij.openapi.util.TextRange
+import org.jetbrains.kotlin.idea.fir.api.fixes.diagnosticFixFactory
+import org.jetbrains.kotlin.idea.frontend.api.fir.diagnostics.KtFirDiagnostic
+import org.jetbrains.kotlin.idea.intentions.AbstractAddAccessorsIntention
+import org.jetbrains.kotlin.psi.KtProperty
+
+class AddAccessorsIntention(addGetter: Boolean, addSetter: Boolean) :
+    AbstractAddAccessorsIntention(addGetter, addSetter) {
+    override fun applicabilityRange(element: KtProperty): TextRange? = element.textRange
+
+    companion object {
+        val addAccessorsToUninitializedProperty = diagnosticFixFactory<KtFirDiagnostic.MustBeInitializedOrBeAbstract> { diagnostic ->
+            // FE1.0 logic in org.jetbrains.kotlin.idea.intentions.AddAccessorsIntention has additional defensive checks to ensure the
```

Comment:
I think you meant to put this comment in `applicabilityRange()` above

Reply:
I actually intended to put it here since if we are going to have the same logic as in FE1.0. They would go here because we cannot run resolve logic on EDT thread. But I do see why it may help if I can move this comment in `applicabilityRange`. Let me do that.

Reply:
Or we could also remove the comment altogether. I don't think we need to reference this particular difference from FE1.0 since it doesn't change the behavior.

Reply:
I think having it here is helpful and I don't think it would cause any confusion. Let's keep it as it is.